backend/src/services/coordinatorService.js

```
@@ -54,6 +52,37 @@ const {
+  const disaster = await repo.findOne({
+    where: { disaster_id: disasterId },
+    relations: ['coordinator', 'coordinator.user']
+  });
+  if (!disaster) throw new Error('Disaster not found');
+  if (disaster.coordinator.user.userId !== coordinatorId) {
+    throw new InvalidCoordinatorActionError('Not authorized');
+  }
+
+  Object.assign(disaster, updates);
+  const saved = await repo.save(disaster);
+  delete saved.coordinator;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)